VirtoCommerce.CatalogModule.Web/Module.cs

```diff
@@ -203,7 +205,8 @@ public override void PostInitialize()
                         nameof(ExportableProduct.ReferencedAssociations),
                         nameof(ExportableProduct.Outlines),
                         nameof(ExportableProduct.Images)))
-                    .WithTabularMetadata(typeof(ExportableProduct).GetPropertyNames()));
+                    .WithTabularMetadata(typeof(ExportableProduct).GetPropertyNames())
+                    .WithAuthorizationHandler(_container.Resolve<CatalogExportSecurityHandler>()));
```

Comment:
Need to add permission auth here too
```
.WithPermissionAuthorization(CatalogPredefinedPermissions.Export, CatalogPredefinedPermissions.Read)
```, scoped are additional checks.

Also you should add handler auth to full data source
```
.WithAuthorizationHandler(_container.Resolve<CatalogExportSecurityHandler>()))
```